examples/examples_util.h

```diff
@@ -17,7 +17,9 @@
  */
 
 #if defined(_WIN32)
+#define WIN32_NO_STATUS
 #include <windows.h>
+#undef WIN32_NO_STATUS
 #include <ntstatus.h>
```

Comment:
This is so obscure. Perhabs add a comment here.

I found https://www.mschaef.com/windows_h_is_wierd

Reply:
Mind suggesting a good wording for a comment?